app/models/internal/TimeSeriesResult.java

```diff
@@ -40,8 +53,18 @@
      *
      * @author Brandon Arp (brandon dot arp at inscopemetrics dot com)
      */
+    @JsonTypeInfo(
+            use = JsonTypeInfo.Id.NAME,
+            defaultImpl = DefaultTimeSeriesResult.Query.class,
+            property = "@type"
+    )
+    @JsonSubTypes(
+            @JsonSubTypes.Type(
+                    name = "default",
+                    value = DefaultTimeSeriesResult.Query.class
+            )
+    )
     interface Query {
-        @JsonProperty("sample_size")
```

Comment:
I believe this was as it is to deserialize a kairosDB query directly into this model.

That mapping is handled manually in #405.